### DISCUSSION TITLE
I received an error, but I don't know how to solve it:step  resolution Failed

### DISCUSSION THREAD

**Discussion by yordyi** [zed-industries/zed#18528] [Feature Requests]
I don't know what the reason is.
<img width="455" alt="image" src="https://github.com/user-attachments/assets/ba813c6e-f05f-4a1b-b994-4b8d92f1a923">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
